moped-editor/src/views/projects/projectView/ProjectComponentsMapEdit.js

```diff
@@ -360,11 +363,6 @@ const ProjectComponentsMapEdit = ({
     // Generate output, clean up & return
     return (
       [...insertionList, ...removalList] // Mix both insertion and removal list
-        // For each subcomponent, mark for deletion (status_id = 0) if component_subcomponent_id has a number
-        .map(subcomponent => ({
-          ...subcomponent,
-          status_id: isNaN(subcomponent?.component_subcomponent_id) ? 1 : 0,
-        }))
```

Comment:
This iteration is no longer need since we add `is_deleted: true` when creating the `removalList` array above.